ckanext/canada/tables/ati.yaml

```diff
@@ -134,6 +134,9 @@ resources:
     description:
       en: The number of pages released in response to the completed Access to Information request. If no records were released, enter a value of “0”.
       fr: Le nombre de pages publiées en réponse à la demande d’accès à l’information. Si aucun enregistrement n’a été publié, entrez la valeur « 0 ».
+    validation:
+      en: The number of pages released must not be negative
+      fr: Le nombre de pages publiées ne doit pas être négatif
```

Comment:
need to define an `excel_error_formula` so that errors are highlighted in the template. See the month column for an example.